### ISSUE TITLE
Is the DazToC4D Bridge compatible with R25?

### ISSUE THREAD

**Issue by rbrandford** [daz3d/DazToC4D#29]
I haven't been able to find a straightforward answer to this question. Is the DazToC4D Bridge compatible with R25?

**Comment by danielbui78** [comment]
Yes.  It will work with R25.  Please let me know where you were checking, so I may update those places with more clear information.  Thanks!

**Comment by danielbui78** [comment]
Did you read the instructions yet?  The menus have changed: https://github.com/daz3d/DazToC4D#3-how-to-use

**Comment by rbrandford** [comment]
I have read and followed the instructions. Still no luck. How do I uninstall the bridge (both the DAZ and C4D parts) and start again? I installed it through DAZ Central and cannot find a way to remove it.

**Comment by danielbui78** [comment]
In Daz Central main screen, click the "Launch" link underneath DazToC4D.  This opens up the Daz To C4D page.  When you hover over the "Open" button, you will see a down arrow.  Click the down arrow and you will see an Uninstall option.

**Comment by rbrandford** [comment]
Problem resolved. Thanks for your help.
